kubernetes/celestia-local/scripts/init-celestia-appd.sh

```diff
@@ -2,6 +2,9 @@
 
 set -o errexit -o nounset
 
+# start with clean config directory.
+rm -rf "$home_dir/config/*"
```

Comment:
Also investigating this right now because I need to test a celestia jsonrpc client.

There is something fishy going on because if one runs `docker run -it --rm --entrypoint /bin/sh ghcr.io/celestiaorg/celestia-app:v1.0.0-rc5`, then `/home/celestia` is in fact not populated.

At this moment I don't understand why in k8s the homedir is prepopulated when this script is executed, while when running the container image directly through docker with the entrypoint replaced it is not.

Specifying `command: [/scripts/init-celestia-appd.sh]` should be equivalent to overriding `ENTRYPOINT` in docker.